content/data-management-plan.md

```diff
@@ -0,0 +1,28 @@
+# Data Management Plan (DMP)
+
+Data management plans help structure data handling throughout a research project. Simply put, data management plans ask the following guiding questions:
+  * Where can collaborators find the data?
+  * Is the data accessible to others?
+  * Is the data interoperable?
+  * Is the data reusable?
+
+Walking through these questions for a research project can help identify potential problems. For example, it might become clear that only one person has access to the data an entire project depends on. Such a situation can easily result in data loss (a project with a [bus factor](https://en.wikipedia.org/wiki/Bus_factor) of one). If that person has extensive back-up plans to prevent data loss, the problem is mitigated somewhat.
+
+Funding agencies increasingly require data management plans, because they heavily invest in research projects and want to maximize their value for money. But data they are super helpful to you as an individual. 
+
+One way to create a data management plan is to walk through the various questions above and think of questions specific to your field of research or legal jurisdictions. For example, in the Netherlands researchers are mandated to disclose (potential) data leaks within 24h, which extends to say a lost USB stick in the train. If personal data is being transferred on USB sticks, making a data management plan could identify this risk and search for ways to mitigate this risk (for example with [Full Disk Encryption](full-disk-encryption.md.html))
+
+Many pages already exist that provide examples of such plans. For example, the [University of Minnesota](https://www.lib.umn.edu/datamanagement/DMP/example) provides a set of field specific examples. There is also the [DMPTool](https://dmptool.org/) that provides you with a web application to make your data management plan. It is worthwhile to search for additional examples dependent on the potential requirements from a funding agency, or from the institute itself. 
+
+Data management plans are the biggest help to yourself, providing a fundamental project management tool to identify key strengths and issues in how data is handled throughout a project.
+
```

Comment:
Should we link to licensing/copyright/data protection on this page somewhere?

Reply:
I added one to licensing now, because there's nothing on data protection yet. Should definitely add something about that at some point!

Reply:
+1 for data protection!